### DISCUSSION TITLE
Homing failed

### DISCUSSION THREAD

**Discussion by marnheus** [Jyers/Marlin#684] [General Discussion]
Hello, I just installed the E3V2-ManualMesh-5x5-v4.2.2.bin precompiled firmware, but when I'm trying to home, nothing moves, and after a short while I get an error screen saying that the homing failed.

Anyone ever saw that? And any idea on how to fix it?

**Comment by marnheus** [comment]
That's actually fixed, I've reflashed it and now it works. Maybe I was not using the correct version

**Reply by nmdemarco** [reply]
I had the same problem. Solved by flashing the "default" precompiled firmware ` E3V2-Default-v4.2.2-v1.3.5.bin`

I had tried the 5x5 manual mesh (no BL probe), and no motion would work. Still learning about the options.
